runtime/src/lib.rs

```diff
@@ -159,6 +160,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     impl_version: 0,
     apis: RUNTIME_API_VERSIONS,
     transaction_version: 5,
+    state_version: 1,
```

Comment:
use state_version: 0，1 means use new trie structure, may not very stable.

Reply:
substate 0.9.16 use 1, i think it is stable already.
https://github.com/paritytech/substrate/blob/polkadot-v0.9.16/bin/node/runtime/src/lib.rs#L129